DISCOVER/_templates/language-switcher.html

```diff
@@ -0,0 +1,19 @@
+<div class="dropdown">
+  <button class="btn btn-sm btn-outline-secondary dropdown-toggle" type="button" id="language-switcher-button"
+    data-bs-toggle="dropdown" aria-expanded="false">
+      {{ language_names[current_language] }}
+  </button>
+  <ul class="dropdown-menu" aria-labelledby="language-switcher-button">
+    {% for item in languages %}
+      {% if item.language and item.url %}
+        <li>
+          <a class="dropdown-item{% if current_language == item.language %}active{% endif %}" 
+              href="{{ item.url }}">
```

Comment:
The link should point to that same page in a different language, not to the homepage in the different language. We can construct the url from a `{{ baseurl }}/` (that should be added to the html_context, in our case it would be `https://discover-cookbook.numfocus.org`), then `{{ current_version }}/` (already in html_context), `{{ item.language }}/` and then the [pagename](https://www.sphinx-doc.org/en/master/development/html_themes/templating.html#pagename) added by sphinx to the html_context and updated for each page automatically